src/sentry/rules/processor.py

```diff
@@ -274,6 +284,9 @@ def activate_downstream_actions(
                 continue
 
             action_inst = action_cls(self.project, data=action, rule=rule)
+            if not isinstance(action_inst, EventAction) and not hasattr(action_inst, "after"):
```

Comment:
`not hasattr(action_inst, "after")` is my workaround for tests in `test_post_process` that use a mock action; not sure if this can be improved